app/auth/templates/auth/login.html

```diff
@@ -98,5 +97,5 @@ <h1 class="h2 text-center">Welcome back!</h1>
 
     </div>
 
-
+  </div>
```

Comment:
the ` </div>` here doesn't seem to have an opening one.

Reply:
I checked several times but PyCharm still complaints a missing closing div tag. It's weird, isn't it?